.pullapprove.yml

```diff
@@ -39,3 +39,18 @@ groups:
       enabled: true
       approve_regex: '^qa-passed'
       reject_regex: '^qa-failed'
+
+  docs-team:
+    conditions:
+      files:
+        - "documentation/*"
+        - "*.md"
+        - "*.rst"
+    required: 1
+    users:
+      - rcaballeromx
+      - mltullis
```

Comment:
Great. But Mike isn't part of the 01org organisation yet, so cannot approve doc changes.